packages/ui/src/components/ChannelListItem/index.tsx

```diff
@@ -52,7 +52,10 @@ export default function ChannelListItem({
           {model.lastPost && <ListItem.Time time={model.lastPost.receivedAt} />}
           {model.unread?.count && model.unread.count > 0 ? (
             <ListItem.Count>{model.unread.count}</ListItem.Count>
-          ) : null}
+          ) : (
+            // Add a spacer to keep the layout consistent
+            <View height={23} />
```

Comment:
If we use `opacity` or `visibility` here, either adding a prop to `ListItem.Count` or creating a wrapper component around it, we can do away with the constant by keeping it there, but hidden.

Reply:
Cool, cleaned up a bit of the display logic as well.